src/controller/MovieController.php

```diff
@@ -82,4 +97,28 @@ public function archiveMovie(int $movieId): array {
         }
         return ['success' => true];
     }
+

```

Comment:
The method a few lines above 'deleteMovie(int movieId)' seems to be unused. It is being referenced from the routes.php file, at line 514, but we're never truly calling that route from anywhere, since we only archiving, not deleting. I believe you forgot to delete them :)